### PR TITLE
Pongo update

### PR DESCRIPTION
**Pull request by apragsdale**
So I got around to doing some more digging on the orangutan genome assemblies. Even though there are multiple Pongo species, it looks like all genomic studies use a Pongo abelii (Sumatran) reference genome (typically Susie, i.e. PonAbe2) to map reads, even for Pongo pygmaeus. So I think it makes sense to keep a single species, and use PonAbe2 as the genome build. I've gone through and changed names (e.g. PonPyg to PonAbe), and updated some citation and genome information. (see #354, #342 for related issues)

I also have permission to upload the two genetic maps, one each for Pongo abelii and pygmaeus. The map info is included in this PR, but I'm not exactly sure what needs to be done to get these maps in the cache (esp. in light of #335).

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Thanks @apragsdale, looks good. As a temporary workaround I've pushed a change in #367 which removes Pongo from the catalog so we can take our time fixing this.

The updates here are excellent and should be used to define the PonAbe species. However, it's not clear how we should deal with the multispecies model here. Whose genome should this be simulated with?

This might require a redesign and refactor.

**Comment by apragsdale** [comment]
> Thanks @apragsdale, looks good. As a temporary workaround I've pushed a change in #367 which removes Pongo from the catalog so we can take our time fixing this.

Makes sense, especially if a refactor is necessary.

> The updates here are excellent and should be used to define the PonAbe species. However, it's not clear how we should deal with the multispecies model here. Whose genome should this be simulated with?

I'm not sure what the right general solution for this problem would be. But for orangutan, even though there are two or three species, genomic studies map reads from each species to a common reference genome, either ponAbe2 or ponAbe3, which comes from a Sumatran orangutan, and all the genome attributes, such as the recombination map, chromosome lengths, etc, correspond to PonAbe.

So if we think of our "species" as "genome assemblies + attributes", I don't think there is an inconsistency in this case. If there also happened to be a genome assembly from a Bornean orangutan, then I think it would warrant its own "species", and we could duplicate the demographic model there, so you would be able to run the simulation using either assembly. Maybe a solution to avoid duplicated code would be to have a "shared_models.py" file, and each genome assembly/species that uses one of those shared models would import the model it needs and add it to the species using `_species.add_demographic_model`?

**Comment by jeromekelleher** [comment]
All sounds sensible to me, thanks @apragsdale. A pragmatic definition of what we mean by species which we document clearly would go a long way to clearing things up.

**Comment by andrewkern** [comment]
hey @apragsdale lets bash this out while you are visiting UO

**Comment by apragsdale** [comment]
I don't know if it would be possible, but getting the Pongo species and demographic model wrapped up before we resubmit would be really great. As it stands now, I have two recombination maps for the two Orang species in hand, both mapped to the standard Pongo abelii genome build (see #342).

I'd propose we file both maps under this species/genome build, which avoids dealing with any multi-species issues for now. I think it's becoming clear in any case that we think of our "species" as linked to a specific genome build, so this would fit under that interpretation. I'll need to update a bit and rebase, but I think it's really close.

**Comment by apragsdale** [comment]
Not sure why the travis CI is failing.. but this is almost ready to go. The recombination maps are also in the HapMap format. I might need some help figuring out where to get those uploaded. Am I forgetting anything else we need to clear up for the Orangs?

**Comment by jeromekelleher** [comment]
LGTM, thanks @apragsdale. @petrelharp, @grahamgower anyone want to take a look here to sort out our Orangs?

(Travis fail just needs a rebase)

File: stdpopsim/catalog/pongo_abelii.py [hunk 1]
```diff
@@ -20,36 +20,36 @@
 # List of chromosomes.
 
 # length information can be found here
-# <http://https://www.ncbi.nlm.nih.gov/genome/325>
-# Orangutan Jan. 2018 (Susie_PABv2/ponAbe3) Browser Sequences
-# Note: no chrY
-# Average recombination rate estimated in Locke et al (doi:10.1038/nature09687)
-# which reports 0.95 +/- 0.72 cM/Mb
+# <http://https://www7.ncbi.nlm.nih.gov/genome/325>
```
Comment (grahamgower) [inline stdpopsim/catalog/pongo_abelii.py:23]: "http://" is duplicated here, and I had to remove the 7 from `www7` to get the orangutan page.

Comment (apragsdale) [reply stdpopsim/catalog/pongo_abelii.py:23]: Thanks Graham, not sure how that got mangled. Will fix.

**Comment by apragsdale** [comment]
Thanks @ndukler for uploading the maps. I'm running into some issues passing the tests for downloading, getting

`ValueError: Tarball format error: member rec_map.PA not a file`.

Is there a trick to getting this to work, or did I perhaps mess something up with the recombination map files I shared with you?

**Comment by ndukler** [comment]
I don't know. It worked fine for the other species and it looks like the URL is correct.

**Comment by apragsdale** [comment]
There's probably some issue with how I compressed the files? I was kind of guessing at exactly how it needs to be formatted for the upload. I don't think that information is documented currently.

**Comment by ndukler** [comment]
I'll look into it

**Comment by ndukler** [comment]
I've got a suspicion it's the `.` in the filenname. Let me rename it to `rec_map_PA.tar.gz` and  `rec_mapPP_.tar.gz`. Please change the urls to reflect this

**Comment by apragsdale** [comment]
Yeah, I think I see the issue - the recombination maps are within a directory in the tar.gz. Let me re-compress this and reshare it with you

**Comment by apragsdale** [comment]
Ok @ndukler I just replaced the maps in that Dropbox I shared with you (they are named NaterPA_PonAbe2.tar.gz and NaterPP_PonAbe2.tar.gz). I think those should work. Let me know if something seems off. And thanks for helping out with this.

**Comment by ndukler** [comment]
And they're up

**Comment by apragsdale** [comment]
Yessss all checks have passed. Thanks a bunch Noah!

**Comment by ndukler** [comment]
I literally just clicked upload a few time :) Thanks for everything you did!

**Comment by grahamgower** [comment]
Awesome, thanks folks! @apragsdale, could you rebase and squash please?

**Comment by apragsdale** [comment]
Yep, all set @grahamgower. Thanks all, really happy to get our Orangutans fixed up before the resubmission.

**Comment by grahamgower** [comment]
Merged! Thanks @apragsdale. There doesn't seem to be an Orangutan-relevant reaction emoticon... Might need to send a github a feature request about that!

**Pull request merged**